app/code/Magento/Review/Api/GetProductRatingSummaryInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Review\Api;
+
+/**
+ * Interface to get product rating summary
+ */
+interface GetProductRatingSummaryInterface
+{
+    /**
+     * Returns the product rating summary
+     *
+     * @param int $productId
+     * @param int $storeId
+     *
+     * @return int Rating summary
+     */
+    public function execute(int $productId, int $storeId = 0): int;
```

Comment:
Based on https://devdocs.magento.com/guides/v2.3/coding-standards/technical-guidelines.html, service contracts should be implemented as batch operations.

Reply:
probably it's better not to have contracts parameterized with default values